### ISSUE TITLE
Clarity on some examples

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/website#331]
Some of the examples assume that you generated a Full project, but if your first time with Lucky was with an API app, then some of the examples might be confusing.

If we mention `BrowserAction`, we can add a note mentioning that it's for Full apps, and API would use `ApiAction`.

Ref: https://github.com/luckyframework/lucky/issues/1207#issuecomment-648440329

We should probably also mention in the routing that you can only use 1 route per action. Technically you can use more than 1, but that will lead to some weird side effects, so we should just say only use 1. Let the multi route stuff for more of an expert escape hatch type deal.... I think eventually that will just raise an error and just say "no" 😛

**Comment by paulcsmith** [comment]
Yeah I agree on mentioning one route per action more clearly! And also agree on raising at compile-time if given more than one route in an action
